backend/typescript/middlewares/validators/authValidators.ts

```diff
@@ -40,7 +40,7 @@ export const registerRequestValidator = async (
   if (!validatePrimitive(req.body.password, "string")) {
     return res.status(400).send(getApiValidationError("password", "string"));
   }
-  if (!validatePrimitive(req.body.password, "string")) {
+  if (!validatePrimitive(req.body.phoneNumber, "string")) {
```

Comment:
should we change this back?

Reply:
so this one was supposed to be phoneNumber, the if statement before checks for password

Reply:
ohhh right ok